### DISCUSSION TITLE
User ID and Group ID adjustment

### DISCUSSION THREAD

**Discussion by stormbyte** [craftcms/docker#37] [Ideas]
### Description

Not all servers allow chmod of files that are not owned by the current user.
When the image requires the files from the host but inside the image there is a forced userID en GroupID of 33:33
It will create permission issues.

Allowing the www-data user to change userID in the container will give an option to match de operating user ID

**Comment by stormbyte** [comment]
https://github.com/craftcms/docker/pull/28

**Comment by sebimalwieder** [comment]
I guess I'm running into the same issues when setting this up on an ubuntu host machine. On my Mac, users and file ownerships doesnt sync up. But on ubuntu i run into permissions problems like `/app/storage is not writable` when running the docker-compose example.
